frontend/pages/api/user-profile-data.js

```diff
@@ -0,0 +1,35 @@
+import { Web3Storage, File } from 'web3.storage';
+
+export default async function handler(req, res) {
+    if (req.method === 'POST') {
+        return await storeUserData(req, res);
+    } else {
+        return res
+            .status(405)
+            .json({ message: 'Method not allowed', success: false });
+    }
+}
```

Comment:
So we can secure this endpoint behind the sign with ethereum authentication, the handler can be wrapped with the `withIronSessionApiRoute ` util...
https://github.com/interplanetaryfonts/interplanetaryfonts.xyz/blob/200375117838dad8b18fa25c5fa21644f4df8353/frontend/pages/api/upload-font.js#L82
```suggestion
import { Web3Storage, File } from 'web3.storage';
import { withIronSessionApiRoute } from 'iron-session/next';

import ironOptions from '../../config/ironOptions';

async function handler(req, res) {
    if (req.method === 'POST') {
        return await storeUserData(req, res);
    } else {
        return res
            .status(405)
            .json({ message: 'Method not allowed', success: false });
    }
}

export default withIronSessionApiRoute(handler, ironOptions);
```

Reply:
Done on 5249db2